### ISSUE TITLE
Item 21 python 2 example needs to use floating point numbers

### ISSUE THREAD

**Issue by bslatkin** [bslatkin/effectivepython#28]
From this part, all of the numerators inputted need to be floats.

```
# Python 2
def safe_division_d(number, divisor, **kwargs):
    ignore_overflow = kwargs.pop('ignore_overflow', False)
    ignore_zero_div = kwargs.pop('ignore_zero_division', False)
    if kwargs:
        raise TypeError('Unexpected **kwargs: %r' % kwargs)
    # ...
Now, you can call the function with or without keyword arguments.
safe_division_d(1, 10)
safe_division_d(1, 0, ignore_zero_division=True)
safe_division_d(1, 10**500, ignore_overflow=True)
Trying to pass keyword-only arguments by position won’t work, just like in Python 3.
safe_division_d(1, 0, False, True)
>>>
TypeError: safe_division_d() takes 2 positional arguments but 4 �were given
Trying to pass unexpected keyword arguments also won’t work.
safe_division_d(0, 0, unexpected=True)
>>>
TypeError: Unexpected **kwargs: {'unexpected': True}
```

**Comment by bslatkin** [comment]
Fixed in example code in f892cd116d27605c57c625be3b79e317759365cb

**Comment by bslatkin** [comment]
Will be fixed in the next printing.
